content/docs/reference-glossary.md

```diff
@@ -0,0 +1,101 @@
+---
+id: glossary
+title: Glossary of React Terms
+layout: docs
+category: Reference
+permalink: docs/glossary.html
+
+---
+
+## Single Page Application
+A single page application is an application that loads a single HTML page and all the necessary assets (CSS, Images & JavaScript) required for the application to run. Any interactions with the page or subsequent pages do not require a round trip to the server which means the page is not reloaded.
+
+
+## ES6/ES2015/ES7/ES2016/ES8/ES2017
+ These acronyms all refer to the most recent versions of the ECMAScript Language Specification standard, which the JavaScript language is an implementation of. It includes many additions to the previous versions such as: arrow functions, classes, template literals, `let` and `const` statements. ES8/ES2017 was finalized in June 2017 and includes async/await functionality. 
```

Comment:
What do you think about linking externally for details? Maybe something like...

> ## ES6/ES2015, ES7/ES2016, ES8/ES2017
> These acronyms all refer to recent versions of the ECMAScript Language Specification standard, which the JavaScript language is an implementation of. They include many additions to the previous versions such as: arrow functions, classes, template literals, `let` and `const` statements. You can learn more about specific versions [here](https://en.wikipedia.org/wiki/ECMAScript#Versions).

Reply:
Great -- will add